Sources/web3swift/Transaction/AbstractEnvelope.swift

```diff
@@ -0,0 +1,133 @@
+// Package: web3swift
+// Created by Alex Vlasov.
+// Copyright © 2018 Alex Vlasov. All rights reserved.
+//
+// Support for EIP-2718 by Mark Loit 2022
+
+import Foundation
+import BigInt
+
+/// Enumeration for supported transaction types
+/// .legacy for untyped and type 0 transactions EIP155 and older
+/// .eip2930 for type 1 transactions conforming to EIP2930
+/// .eip1559 for type 2 transactions conforming to EIP1559
```

Comment:
Also as a suggestion: maybe it is worth moving each of these swift doc lines to its respective case?

Reply:
Good suggestion. I had originally thought of doing that, but the XCode shortcut didn't want to let me add one there, so I left it. But I will manually add them in there.